azurelinuxagent/ga/update.py

```diff
@@ -113,6 +113,9 @@ def __init__(self, vm_status=None):
     def __eq__(self, other):
         return self.summary == other.summary
 
+    def __ne__(self, other):
+        return not (self.summary == other.summary)
```

Comment:
Might as well make the change that you suggested on IM (i.e. `not(self == other)` here too?

Reply:
Yes, but I'm doing that on a separate PR to keep a cleaner history